mirdata/datasets/medleydb_melody.py

```diff
@@ -46,6 +46,8 @@
     {}
 """
 
+LICENSE_INFO = "http://creativecommons.org/licenses/by-nc-sa/4.0/."
```

Comment:
nit - for the other creative commons licenses we're writing the name of the license in text but this one is a link. Maybe we want them all to have links?

Reply:
Mmmm.. I used links for those that only provide links in their websites. I'm a bit afraid of not matching exactly the license if I add links to the other ones. Want me to change this link to the name of the license instead?

Reply:
We don't take responsibility for accuracy so it's not horrible if it's inconsistent. I'm fine either way though (to leave as is or to change it)

by the way, there are lists like this with badges - https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba

we'd talked about adding a column to the datasets table with the license badge. is it worth doing here?

Reply:
I'll leave it like is then.

Will take a look at the table now

Reply:
@genisplaja I see we have a single entry for Saraga. Should we have one for hindustani and another for carnatic?

Reply:
Sorry for the late reply @magdalenafuentes!! I think we should have two entries because there are some differences between the available audio and annotations. I will push the new rows in this PR if you are OK with it.

Reply:
Sure! Go ahead

Reply:
Done!!